torch/distributed/nn/api/remote_module.py

```diff
@@ -155,18 +155,22 @@ def __init__(
         args = args if args is not None else ()
         kwargs = kwargs if kwargs is not None else {}
 
-        self.on = on
+        [self.on, self.device] = remote_device.split("/")
```

Comment:
Let's add a helper method `_parse_remote_device` under `torch.distributed.rpc.utils` which we can later reuse in other parts like RemoteDevice etc.

Reply:
Created a utils.py under torch/distributed/rpc.